### DISCUSSION TITLE
iOS - Send through "Share Menu"?

### DISCUSSION THREAD

**Discussion by mensa84** [localsend/localsend#920] [Discussion (English, International)] [answered]
Hello,

is it possible to add localsend to the "Share Menu" of iOS?
Or is it only possibility to send files to open the localsend app and select files to send?

**Comment by Tienisto** [comment, marked as answer]
It should be already possible but only on newer iOS versions.

**Reply by mensa84** [reply]
I found it now, thank you!

**Reply by ekohilas** [reply]
Hey @Tienisto 
Could you please be more specific on what "newer" iOS versions means?
I can't seem to find the option.

**Reply by hazzuk** [reply]
FYI, LocalSend might not immediately show in your iOS share sheet. It did for my iPad, but I first had to restart my iPhone before it appeared in the list of available apps. Which you can get to by swiping towards the right of the apps list and selecting the three dotted icon 'More'.

**Reply by ekohilas** [reply]
Hey @SirHazza, thanks for your reply!
After 2 months and many restarts I'm yet to see it show in my iOS share sheet, even after going into 'More'
I suspect the "newer" versions is an issue here, and I'm keen to better understand from @Tienisto why that might be the case.

**Reply by Tienisto** [reply]
I am not a native iOS user but I assume that you need iOS 16 or 17. I have an iPhone 5S for testing purposes and it never worked because the iOS version is likely too old
